rsciio/mrc/_api.py

```diff
@@ -269,24 +323,69 @@ def file_reader(
 
     %s
     """
+    if metadata_file == "auto":
+        if "movie" in filename:
+            try:  # DE movie
+                dir_name = os.path.dirname(filename)
+                base_name = os.path.basename(filename)
+                split = base_name.split("_")
+                unique_id = "_".join(split[:2])
+                if len(split) > 3:  # File Suffix
+                    suffix = "_".join(split[2:-1])
+                else:
+                    suffix = ""
+                metadata = glob.glob(dir_name + "/" + unique_id + suffix + "_info.txt")
+                virtual_images = glob.glob(
+                    dir_name + "/" + unique_id + suffix + "_[0-4]_*.mrc"
+                )
+                external_images = glob.glob(
+                    dir_name + "/" + unique_id + suffix + "_ext[1-4]_*.mrc"
+                )
```

Comment:
Is there a reference available somewhere?

I am wondering if it would be useful to document to have all this documented in https://hyperspy.org/rosettasciio/file_specification/index.html?